x-pack/plugins/event_log/scripts/mappings.js

```diff
@@ -173,9 +173,6 @@ exports.EcsCustomPropertyMappings = {
                       prepare_rule_duration_ms: {
                         type: 'long',
                       },
-                      prepare_to_run_duration_ms: {
-                        type: 'long',
-                      },
```

Comment:
As we removed loadIndirectParams method and the timer metric in it, `prepare_to_run_duration_ms` can be removed `prepare_rule_duration_ms` will collect the total time.

Reply:
Note that it can be removed since it was just added a few days ago in https://github.com/elastic/kibana/pull/178044

Once we ship a field in the mappings in a release, we need to think really hard about removing them.  Not a problem in this case though ...